unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -3,7 +3,13 @@ import React from 'react'
 import styled from 'styled-components'
 
 export const LayoutButton = ({href, title, children}) => (
-  <Button href={href}>
+  <Button
+    href={href}
+    backgroundColor={'var(--grey)'}
```

Comment:
I love this change and use of vars - but I wonder whether this could be done via styled component?

Reply:
What do you mean? This is actually done with styled components? We basically pass this as props so that we can re-use most of the `Button` ?